src/card-validator/card-validator.spec.ts

```diff
@@ -2970,4 +2970,26 @@ describe('card-validator', () => {
       expect(result.card?.niceType).toBe(card.issuer)
     })
   })
+
+  it('should be a correctly mask for a card number', () => {
+    cards.forEach((card) => {
+      const result = valid.number(card.cardNumber)
+
+      if (!result.card?.type) return
```

Comment:
Aqui a gente poderia trocar para um assert, esse IF pode mascarar algum problema.
O que acha?

```suggestion
      expect(result.card?.type).toBeDefined()
```